helix-term/src/keymap.rs

```diff
@@ -137,20 +137,28 @@ impl KeyTrieNode {
     }
 
     pub fn infobox(&self) -> Info {
-        let mut body: Vec<(&str, Vec<KeyEvent>)> = Vec::with_capacity(self.len());
+        let mut body: Vec<(&str, BTreeSet<KeyEvent>)> = Vec::with_capacity(self.len());
         for (&key, trie) in self.iter() {
             let desc = match trie {
                 KeyTrie::Leaf(cmd) => cmd.doc(),
                 KeyTrie::Node(n) => n.name(),
             };
             match body.iter().position(|(d, _)| d == &desc) {
-                // FIXME: multiple keys are ordered randomly (use BTreeSet)
-                Some(pos) => body[pos].1.push(key),
-                None => body.push((desc, vec![key])),
+                Some(pos) => {
+                    body[pos].1.insert(key);
+                }
+                None => {
+                    let mut keys = BTreeSet::new();
+                    keys.insert(key);
+                    body.push((desc, keys));
+                }
```

Comment:
```suggestion
                None => body.push((desc, BTreeSet::from([key]))),
```
We can probably just do this.